development/requirements2.0.0-beta.2.txt

```diff
@@ -0,0 +1 @@
+nautobot-bgp-models @ git+https://github.com/nautobot/nautobot-plugin-bgp-models@v2
```

Comment:
I ripped out these conditional requirement files in #242. Do we plan on keeping separate 1.X/2.X branches? If so, we should just change the standard requirements file.

Reply:
Yes, there will be separation between 1.X and 2.X.
I have seen this PR you mentioned above, I'll have to re-base and follow that.